3483/3483_2/bot_start.rb

```diff
@@ -0,0 +1,27 @@
+require 'telegram/bot'
+require 'logger'
+require 'ohm'
+require 'redis'
+require 'active_support/all'
+require_relative 'start.rb'
```

Comment:
https://stackoverflow.com/questions/735073/best-way-to-require-all-files-from-a-directory-in-ruby